internal/api/utils.go

```diff
@@ -752,3 +757,32 @@ func GenerateAndStreamCSV(w io.Writer, recommendationSets []model.Recommendation
 	}
 	return nil
 }
+
+func resolveResponseFormat(acceptHeaderVal string, formatQueryParamVal string) (string, error) {
```

Comment:
From this method, it looks like, you are only consider `format` query param value when `Accept` header is not set or missing or not supported. correct?
That means  `format` query param is being used as fallback and not to override the value whereas in few of the APIs implementation side, query parameters are allowed to override headers explicitly. So, in my opinion, you should clearly document this here - https://github.com/RedHatInsights/ros-ocp-backend/blob/main/openapi.json#L26 to clarify the rationale for including the format query parameter.